torax/_src/geometry/standard_geometry.py

```diff
@@ -17,22 +17,23 @@
 This is a geometry object that is used for most geometries sources
 CHEASE, FBT, etc.
 """
-from collections.abc import Mapping
+
 import dataclasses
 import logging
+from collections.abc import Mapping
+from typing import Any
 
 import chex
 import contourpy
 import numpy as np
 import scipy
-from torax._src import constants
-from torax._src import interpolated_param
-from torax._src.geometry import geometry
-from torax._src.geometry import geometry_loader
-from torax._src.geometry import geometry_provider
-from torax._src.torax_pydantic import torax_pydantic
 import typing_extensions
 
+from torax._src import constants, interpolated_param
+from torax._src.geometry import geometry, geometry_loader, geometry_provider
+from torax._src.torax_pydantic import torax_pydantic
+from torax.torax_imastools.equilibrium import geometry_from_IMAS
```

Comment:
again here load the module not the object https://google.github.io/styleguide/pyguide.html#22-imports